README.md

```diff
@@ -0,0 +1,49 @@
+# zsh-rustup-completion
+
+Oh-my-zsh rustup completion plugin.
+
+Rustup version: 1.11.0 (e751ff9f8 2018-02-13)
```

Comment:
Let's emphasize version with bold, we can now!

Reply:
Done, I'll push